Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM golang:1.16 AS builder
+
+WORKDIR /app
+
+COPY ./ ./
+
+RUN go mod download -x
+
+WORKDIR /app/cmd
+
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix nocgo -o /jokes-api
+
+WORKDIR /app
+
+FROM alpine:latest
+RUN apk --no-cache add ca-certificates
+RUN apk --no-cache add tzdata
+
+WORKDIR /
+RUN pwd
+COPY --from=builder /jokes-api .
```

Comment:
It's not very good to do copy so many times. Production images should contain a low number of lines. You can prepare and copy one folder with one operation